pypiper/pypiper.py

```diff
@@ -461,6 +472,12 @@ def run(self, cmd, target=None, lock_name=None, shell="guess", nofail=False, cle
 		# in your pipeline can depend on this since it won't be run if that command 		# isn't required because target exists.
 		return process_return_code
 
+
+	@staticmethod
+	def _likely_shell(cmd):
```

Comment:
maybe let's call this somethign a bit more descriptive; maybe `_is_likely_shell()` ? or better, `check_shell()` or `shell_check_command()`, since it's not a method on a "Command" object but a static, which is what `likely_shell` seems to fit better with.